src/main/g8/build.sbt

```diff
@@ -14,22 +15,26 @@ lazy val root = (project in file("."))
     // Options used by `native-image` when building native image.
     // https://www.graalvm.org/docs/reference-manual/native-image/
     graalVMNativeImageOptions ++= Seq(
-        "--initialize-at-build-time", // Auto-packs dependent libs at build-time
         "--no-fallback", // Bakes-in run-time reflection (alternately: --auto-fallback, --force-fallback)
         "--no-server", // Won't be running `graalvm-native-image:packageBin` often, so one less thing to break
-        "--static" // Disable for OSX (non-docker) builds - Forces statically-linked binary, compatible with libc (linux)
+        "--static", // Disable for OSX (non-docker) builds - Forces statically-linked binary, compatible with libc (linux)
+        "--initialize-at-build-time=" + Seq( // Auto-packs dependent libs at build-time
+            "$package$.Main",
+            // "some.class.with.long.initialization.Time"
+        ).mkString(",")
```

Comment:
This was LAZY as heck. Sorry for that. Here's the reasoning though: https://www.graalvm.org/reference-manual/native-image/ClassInitialization/#explicitly-specifying-class-initialization